main.js

```diff
@@ -195,6 +206,14 @@ function readIndividualFeed (x) {
                                 let sendung = getShowDetails(titel).show;
                                 let beschreibung = result.rss.channel.item[i].description;
 
+                                rawData.push({
+                                    title: getShowDetails(titel).show,
+                                    description: result.rss.channel.item[i].description,
+                                    station: getShowDetails(titel).station,
+                                    time: getShowDetails(titel).time,
+                                    imgUrl: result.rss.channel.item[i].enclosure && result.rss.channel.item[i].enclosure.url ? result.rss.channel.item[i].enclosure.url : ''
+                                })
```

Comment:
Muss dort nicht ein Semikolon hin?
App-Veyor meckert leider über das Füllen der Raw-Datenpunkte
[AppVeyor Testergebnis](https://ci.appveyor.com/project/Pix---/iobroker-tvspielfilm/builds/31854810/job/jsn7pgqci42abubr)
```suggestion
                                });
```

Reply:
> 
> 
> Muss dort nicht ein Semikolon hin?
> App-Veyor meckert leider über das Füllen der Raw-Datenpunkte
> [AppVeyor Testergebnis](https://ci.appveyor.com/project/Pix---/iobroker-tvspielfilm/builds/31854810/job/jsn7pgqci42abubr)

Jap gehört dahin